goldens/by_regex.in

```diff
@@ -0,0 +1,40 @@
+No capturing group
```

Comment:
Another test case I'm particularly interested is a multi-line one (i.e., utilizing block or group mode). e.g., one FR we had before was sorting by function name when the function signatures might be something like:

```cpp
someReallyLongReturnType<CPPlol, IDK<whatever>>
  myFunc1();

int myFunc2();
```

Which can be done now, as regex would be able to ignore the line breaks and declare a capturing group to match the function name. (The ethics of trying to parse a C++ type with regex is omitted lol.)

Reply:
Done!